import_ldraw.py

```diff
@@ -793,7 +863,9 @@ def create_model(self, context, scale):
                     if bpy.ops.object.mode_set.poll():
                         # Go back to object mode, set origin to geometry
                         bpy.ops.object.mode_set(mode='OBJECT')
-                        bpy.ops.object.origin_set(type='ORIGIN_GEOMETRY')
+                        # When using linked bricks, keep original origin point
+                        if not LinkParts:
+                            bpy.ops.object.origin_set(type='ORIGIN_GEOMETRY')
```

Comment:
Since we use origin to geometry only for non-linked parts, is the Blender default option used instead? I almost want to make that explicit and add a line for linked parts that defines the origin, even if it is the default option.